subprojects/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/ConfigurationCacheBuildOperationsIntegrationTest.groovy

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.configurationcache
+
+import org.gradle.integtests.fixtures.BuildOperationsFixture
+import org.gradle.internal.configurationcache.ConfigurationCacheLoadBuildOperationType
+import org.gradle.internal.configurationcache.ConfigurationCacheStoreBuildOperationType
+import org.gradle.test.fixtures.file.TestFile
+
+class ConfigurationCacheBuildOperationsIntegrationTest extends AbstractConfigurationCacheIntegrationTest {
+
+    def operations = new BuildOperationsFixture(executer, temporaryFolder)
+
+    def "emits no load/store build operations when not used"() {
+        given:
+        withLibBuild()
```

Comment:
Do we really need the lib and app builds here?

Reply:
I want to ensure that there is a single of these operations per build tree, and not one per build

Reply:
might be an implementatoin detail that there is inded a single per build tree, but I want to ensure we don't break this at some point without knowing. The GE side will strongly rely on having a single on these on the whole event stream.

Reply:
I see. In that case it might make sense to have another test where the included build is used only for build logic.

Reply:
done